docs/configuration.rst

```diff
@@ -113,10 +113,6 @@ below:
      - The percentage of events that should be captured (e.g. memory
        allocation). Greater values reduce the program execution speed. Must be
        greater than 0 lesser or equal to 100.
-   * - ``DD_PROFILING_MAX_EVENTS``
```

Comment:
Was this intentional? It still exists and is used in the profiler 🤔 (although with a different default)

Reply:
I didn't want to expose that as we don't really want users to shoot their own foot with it. We can still expose it as a debug option just in case.